tools/build-release-branch.sh

```diff
@@ -107,6 +133,22 @@ function create_new_release_branches {
 		# Create new branch, push to repo
 		git checkout -b $NEW_UNBUILT_BRANCH
 
+		# Updates file target version.
+		read -n1 -p "Would you like to update the version number in files to $TARGET_VERSION? [y/N]" reply
+		if [[ 'y' == $reply || 'Y' == $reply ]]; then
+			# Replace all file contents.
+			sed -r -i "s/Version: .+/Version: ${TARGET_VERSION}/" jetpack.php
+			sed -r -i "s/'JETPACK__VERSION',(\s+)'(.+)'/'JETPACK__VERSION',\1'${TARGET_VERSION}'/" jetpack.php
+			sed -r -i "s/\"version\": \".+\"/\"version\": \"${NPM_TARGET_VERSION}\"/" package.json
+
+			# Commit changed files.
+			git commit -m "update version" jetpack.php package.json
```

Comment:
I sign commits out of absolute habit now. Do we expect all Jetpack committers to have that setup? Could we do `git commit -S -m "....`?

Reply:
I don't know. @dereksmart, is that good for you?

Reply:
We don't do any signature verification, so there is not really any use for `-S`. Are we sure that signing will work for everyone's git setup? With their privacy settings and all?

Reply:
From the [documentation](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work#_everyone_must_sign)

> Everyone Must Sign
Signing tags and commits is great, but if you decide to use this in your normal workflow, you’ll have to make sure that everyone on your team understands how to do so. If you don’t, you’ll end up spending a lot of time helping people figure out how to rewrite their commits with signed versions. Make sure you understand GPG and the benefits of signing things before adopting this as part of your standard workflow.

Reply:
So, my answer is "no". Let's leave it without signatures.

Reply:
Sounds good. Resolving.